### ISSUE TITLE
Rule merging problematic with unsupported selectors/pseudoclasses

### ISSUE THREAD

**Issue by mischnic** [parcel-bundler/parcel-css#3]
```css
.form-input:-moz-placeholder-shown {
  color: black;
}
.form-input:-ms-input-placeholder {
  color: black;
}
.form-input:placeholder-shown {
  color: black;
}
```

is merged into a single rule. But browsers ignore rules with invalid selectors, so the rule won't apply on any browser.

via https://github.com/evanw/esbuild/releases/tag/v0.13.14

**Comment by devongovett** [comment]
Yeah. Need to handle vendor prefixes for pseudo selectors the same way we handle them for declarations as well, e.g. adding and removing according to browser targets.

**Comment by devongovett** [comment]
Should be fixed by f1a074f2a8ead4b4ef22c1eb1b6b23eaa8c9f7a4. Now it checks that all of the selectors in both rules are supported by all targets before merging rules.
